ampersand-select-view.js

```diff
@@ -243,6 +257,20 @@ SelectView.prototype.getOptionValue = function (option) {
     return option;
 };
 
+SelectView.prototype.getOptionDisabled = function (option) {
+    if (Array.isArray(option)) return;
```

Comment:
@kamilogorek The disabled functionality does not currently work for non model items. So if the type is array I think it should simply return here. For items which are an array ( when sending pairs as the options), getOptionValue returns option[0] and getOptionText returns option[1]. option[2] would be undefined. 

We could however extend the functionality for sending arrays to allow a 3rd value which would toggle disabled on the option. e.g.

    options: [ ['a', 'Option A', true], ['b', 'Option B', false], ['c', 'Option C', true] ]